ServerWhitelist.yml

```diff
@@ -1949,12 +1949,6 @@ AllowedPrefixes:
     - https://cdn.discordapp.com/attachments/1177560192192352336/1178355556113731604/SexLab_P_Quick_Scene_Start_Spell_-_4.0.0.zip?ex=65db5ba1&is=65c8e6a1&hm=0afebe8fd8aedc5ce84171cc1b0d649fd62bbe888f4d75c89f95fcdb6c34ab4c& # Sexlab + Quick Spell Inline
```

Comment:
for this file if you got inline permissions put sources to those permissions into the mod and use the following compiler settings to include it with the list or ask for mirror permissions and put it onto the CDN as well (with proof included).
https://wiki.wabbajack.org/modlist_author_documentation/Compilation%20Settings.html

Because this doesn't need to be whitelisted that way+ discord attachment URLs could likely break in the future due to Discord changing how they handle links to files in the chats.